docs/documentation/siddhi-4.0.md

```diff
@@ -46,6 +46,208 @@ by a semicolon `( ; )` as shown in the below syntax.
         ;
 ```
 
+**Performance**
```

Comment:
Can you move these sections to the bottom to the doc, as these are more advanced configurations? It can be under a heading such as "Configuring and monitoring Siddhi Apps"  and each element can be subheadings. Same for the SP wiki.